### DISCUSSION TITLE
Axes max minutes keeps resetting to 10:00

### DISCUSSION THREAD

**Discussion by grabowc** [artisan-roaster-scope/artisan#1429] [General]
We need our axes to show about 12-15:00 in duration. When we try to adjust the setting from 10:00 to 15:00, it changes for a bit, but it eventually returns to 10:00. How do we get this to STAY at 15:00 max duration in Axes setting for all ?
Thanks!

**Comment by roasterdave** [comment]
- Config>> Axes:
- Untick **Auto** 
- Untick **Expand** 
- Enter your desired axes values in both sets of **Min/Max** boxes
- Tick **Lock**

![image](https://github.com/artisan-roaster-scope/artisan/assets/30009312/40c3e868-5c65-4958-8266-92add0a5566e)

**Reply by grabowc** [reply]
Thanks!

What does Expand do if checked?

**Reply by grabowc** [reply]
In fact, where are all the settings explained? I'm happy to read the manual if I can find it!

**Comment by roasterdave** [comment]
`In fact, where are all the settings explained? I'm happy to read the manual if I can find it!`
Thanks for asking!  There is no single place to see **all** the settings explained.  Artisan is community supported open source software and we are always looking for more help with documentation (and everything else).  Many users ask for instructional videos while others want to see written documentation.  Unfortunately our limited resources can't satisfy every need.  We welcome any help with documentation, testing, coding, design, translation, support, training tutorials or anything else.  

Subscribing to [artisan.plus](https://artisan.plus/) is a great way to support continued Artisan development in addition to getting access to excellent inventory management that is tightly integrated with Artisan.  [Donations ](https://artisan-scope.org/donate/)are another way to support Artisan.

Some of the places where you can find information about using Artisan:

- Release notes for each version are available [here](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/ReleaseHistory.md).  
- The [Artisan blog](https://artisan-roasterscope.blogspot.com/) has articles with more depth on the features in each release.  The blog also has articles focused on specific features and on other relevant topics. 
- The [Artisan home page](https://artisan-scope.org/) has a [Quick Start Guide](https://artisan-scope.org/docs/quick-start-guide/) for new users, information on supported Machines and Devices, links to video tutorials and a lot more.
- Artisan itself has Help buttons in many places to show more information.  
- Many input fields show their purpose when the mouse is hovered over them.
- And there is always this discussion board for asking questions.

`What does Expand do if checked?`
Here is what is shown when hovering the mouse over the Expand check box.

![Screenshot - 1_31_2024 , 2_47_40 PM](https://github.com/artisan-roaster-scope/artisan/assets/30009312/740e8c29-58e8-40a5-8757-076c68d0753f)

**Reply by grabowc** [reply]
Thanks so much!

**Reply by MAKOMO** [reply]
There is a full tutorial video just on the Axes dialog: https://www.youtube.com/watch?v=QRAzYkmGYVw
